src/StoreContext/StoreContext.tsx

```diff
@@ -490,8 +497,8 @@ export const StoreProvider = ({
       ]) => {
         const code = codeFromQuery || storedCode || exampleCode;
         const editorPosition = storedPosition;
-        const editorHeight = storedHeight;
-        const editorWidth = storedWidth;
```

Comment:
Should we add any backwards compatibility here? If the stored value was not a percentage, convert it to a percentage as part of the initial load?

If not, what is the UX? Does the user need to resize the editor to get the new behaviour?

Reply:
Fair enough, I've updated it to have backwards compatibility